doc/lightning-delpay.7.md

```diff
@@ -0,0 +1,58 @@
+lightning-delpay -- Command for removing a completed payment
+============================================================
+
+SYNOPSIS
+--------
+
+**delpay** *payment_hash* \[status\] (is not specify the status is complete)
```

Comment:
You have a `partid` as well that is not documented.

Defaults should be described in "DESCRIPTION" section, not in the synopsis.

Reply:
Updaed